gammapy/background/background_estimate.py

```diff
@@ -0,0 +1,85 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from .ring import ring_area_factor
+from ..region import find_reflected_regions
+
+__all__ = [
+    'BackgroundEstimate',
+    'ring_background_estimate',
+    'reflected_regions_background_estimate',
+]
+
+
+class BackgroundEstimate(object):
+    """Container class for background estimate
+
+    This container holds the result from a region based background estimation
+    for one observation. Currently, it is filled by the functions
+    :func:`~gammapy.background.ring_background_estimate` and
+    :func:`~gammapy.background.reflected_regions_background_estimate`. At some
+    points this should be replaced by a more elaborate analysis class.
+
+    Parameters
+    ----------
+    off_events : `~gammapy.data.EventList`
+        Background events
+    off_region : `~gammapy.extern.regions.SkyRegion`
```

Comment:
Put `off_region` before `off_events` in the docstring.
(Always use the same order in the function signature and docstring.)